test/RibbonThetaVault.js

```diff
@@ -2576,10 +2610,25 @@ function behavesLikeRibbonOptionsVault(params) {
         const receipt = await tx.wait();
         const gasFee = gasPrice.mul(receipt.gasUsed);
 
+        await expect(tx)
+          .to.emit(this.vault, "Withdraw")
+          .withArgs(
+            user,
+            BigNumber.from("99500000000"),
+            BigNumber.from("100000000000"),
+            BigNumber.from("500000000")
+          );
+
         await expect(tx)
           .to.emit(this.vault, "ScheduledWithdrawCompleted")
           .withArgs(user, BigNumber.from("99500000000"));
 
+        // Should set the scheduledWithdrawals entry back to 0
+        assert.equal(
+          (await this.vault.scheduledWithdrawals(user)).toString(),
```

Comment:
Ah, I was trying `this.vault.scheduledWithdrawals(user)` before and it was failing. Will that only work for public variables?